esp/scm/ws_dfu.ecm

```diff
@@ -174,8 +174,8 @@ ESPStruct [nil_remove] DFUFileDetail
     [min_ver("1.37"), json_inline(1)] string jsonInfo;
     [min_ver("1.37")] binary binInfo;
     [min_ver("1.38")] string PackageID;
-    [min_ver("1.39")] ESPstruct DFUFilePartition Partitions;
-    [min_ver("1.39")] ESPstruct DFUFileBloom Blooms;
+    [min_ver("1.39")] ESPstruct DFUFilePartition Partition;
```

Comment:
There can only be one PARTITION attribute, but there can be several BLOOM attributes. I'm not sure the code below handles this

Reply:
@richardkchapman for testing, I am trying to create an index with multiple BLOOM attributes using the following ECL code. The eclagent throws an exception: 'Published record size 22 for file thor::fred, does not match coded record size 13'. Please advise how to fix the ECL code.

rec := RECORD
   string1 a;
   string3 b;
   string1 c;
   unsigned d;
END;

boolean useBloom := true : stored('useBloom');

d := dataset('fred', rec, FLAT);

i1 := index(d, {a,b,c,d}, 'kw_ix_bloom_a_b_1');
buildindex(i1, overwrite, BLOOM(a, LIMIT(10), PROBABILITY(0.1)), BLOOM(b, LIMIT(12), PROBABILITY(0.2)));

Reply:
Never mind. The problem is in my data file. I fixed it and just creates an index with multiple BLOOM attributes. @wangkx